packages/tocco-ui/src/FormattedValue/typeFormatterProvider.js

```diff
@@ -64,6 +64,7 @@ export const map = {
   'phone': PhoneFormatter,
   'postcode': StringFormatter,
   'remote': SingleSelectFormatter,
+  'remote-field': SingleSelectFormatter,
```

Comment:
did you change this line intentionally?  doesnt seem to be related to the commit msg

Reply:
and is it needed?

Reply:
It is not. All related fields should be either of type single-remote-field or multi-remote-field.